### DISCUSSION TITLE
Has Any One Tried to Use Marten with Google Cloud SQL?

### DISCUSSION THREAD

**Discussion by jeff-kwak** [JasperFx/marten#2397] [Q&A] [answered]
I'm not sure if the PLv8 extension is required to use Marten, or if that's supported in GCP Cloud SQL. If anyone has, I'd be interested to know if there are any tradeoffs or gotchas.

**Comment by mysticmind** [comment, marked as answer]
plv8 is an optional requirement (plugin) for [plv8 based patch and transformations](https://martendb.io/documents/plv8.html#marten-plv8). You can use Marten without plv8.

**Reply by jeff-kwak** [reply]
Thanks for clarifying that for me. I missed that this was only one feature (a nice one to be sure). I read transforms, and incorrectly thought it applied to Event Sourcing projections.
